internal/ips/request.go

```diff
@@ -86,10 +119,14 @@ func (c *Client) Request() *cobra.Command {
 	requestIPCmd.Flags().StringVarP(&metro, "metro", "m", "", "Code of the metro where the IP Reservation will be created")
 	requestIPCmd.Flags().IntVarP(&quantity, "quantity", "q", 0, "Number of IP addresses to reserve.")
 	requestIPCmd.Flags().StringSliceVar(&tags, "tags", nil, "Tag or Tags to add to the reservation, in a comma-separated list.")
+	requestIPCmd.Flags().IntVarP(&cidr, "cidr", "i", 0, "The size of the desired subnet in bits.")
+	requestIPCmd.Flags().StringVarP(&network, "network", "n", "", "The starting address for this VRF IP Reservation's subnet")
+	requestIPCmd.Flags().StringVarP(&vrfID, "vrfID", "v", "", "VRF UUID")
+	requestIPCmd.Flags().StringVarP(&details, "details", "d", "", "VRF IP Reservation's details")
```

Comment:
Same comment about the short argument. While `-d` is more obviously aligned to `--details`, I don't think `--details` is a heavily used argument so we don't need to squander a short argument assignment on it.

Same for customdata.